lgc/patch/NggPrimShader.cpp

```diff
@@ -827,6 +827,8 @@ void NggPrimShader::constructPrimShaderWithoutGs(Module *module) {
   assert(m_ldsManager->getLdsRegionStart(LdsRegionVertPosData) % SizeOfVec4 == 0);
 
   const bool disableCompact = m_nggControl->compactMode == NggCompactDisable;
+  if (disableCompact)
+    assert(std::make_tuple(m_gfxIp.major, m_gfxIp.minor) >= std::make_tuple(10, 3)); // Must be GFX10.3+
```

Comment:
nit: use `std::tie` instead, it should handle temporaries just fine. It doesn't matter for scalars in this particular case, but from what I've seen it's much more idiomatic.

Reply:
Also everywhere else